### ISSUE TITLE
Preslots do not contain basic KAT items

### ISSUE THREAD

**Issue by Thibugs** [team-gsri/GSRI-Gear#61]
Since we are now using KAT for the medical gameplay, it would be a good idea to add basic items to the preslots.

Basic stuff would be for example an oxymeter, some perfalgan, maybe a guedel tube?

Same thing for the medic preslot, it does not have any KAT stuff right now, and make it nearly unusable.
As discussed last mission, some one familiar with KAT and that know how to use it should already have a slot ready. However when guest are coming to GCAP's mission, they won't be able to use their saved equipment, therefore having at least a basic medic gear should be a nice addition.

**Comment by qay21** [comment]
The question seems to be more about whether an untrained, unprepped guest SHOULD be relied on for medical tasks. And I think the answer is no. Having no medic preslot available is one of the factors inciting team leaders to designate trained medics instead of guests.

As for whether or not to provide some KAT specific items in other preslots, I think it is a bad idea. I can't think of anything else but these 3 situations :
* No immediate threat, and a designated medic is present : medical gear from other teammates is not required
* No immediate threat, but no designated medic nearby : a teammate must perform only very basic actions, such as placing tourniquets, eventually CPR, and most importantly placing unconscious teammates on their side to avoid occlusion/obstruction. Any additionnal gear can and will confuse people and make them click on buttons without knowing.
* Immediate threat : no medical assistance is to be performed until all immediate threat are cleared

To summarize :
* Only trained personnel should be carrying advanced medical gear
* Non-trained personnel should carry only non-confusing, easy to use, no-question-asked items that will not clutter their medical menu/actions and won't make them focus on medical fussing-around instead of providing combat protection

**Comment by Thibugs** [comment]
You are right, giving confusing medical items to non trained personnel will confuse them and distract them from their first task : Protection.

As for the medical preslot itself, if we are not going to use it nor update it, we should remove it to avoid any confusion for guests.

**Comment by Thibugs** [comment]
Preslots removed in #62
